init.lua

```diff
@@ -197,36 +201,41 @@ function xban.get_record(player)
 end
 
 minetest.register_on_prejoinplayer(function(name, ip)
-	local wl = db.whitelist or { }
-	if wl[name] or wl[ip] then return end
-	local e = xban.find_entry(name) or xban.find_entry(ip)
-	if not e then return end
-	if e.banned then
-		local date = (e.expires and os.date("%c", e.expires)
-		  or "the end of time")
-		return ("Banned: Expires: %s, Reason: %s"):format(
-		  date, e.reason)
-	end
-end)
+    local wl = db.whitelist or {}
+    if wl[name] or wl[ip] then return end
 
-minetest.register_on_joinplayer(function(player)
-	local name = player:get_player_name()
-	local e = xban.find_entry(name)
-	local ip = minetest.get_player_ip(name)
-	if not e then
-		if ip then
-			e = xban.find_entry(ip, true)
-		else
-			return
-		end
-	end
-	e.names[name] = true
-	if ip then
-		e.names[ip] = true
-	end
-	e.last_seen = os.time()
+    local e = xban.find_entry(name)
```

Comment:
Please use tabs to indent, like used in the rest of the code.

EDIT: The same goes for `xban.find_entry`. I did not notice that the indent style is now mixed.